Resources/public/less/tables.less

```diff
@@ -124,51 +85,21 @@ table.list {
       background-color: #f9f9f9;
     }
     td, th {
-        border-color: transparent;
+      border-color: transparent;
```

Comment:
If border is transparent, then do we realy need any border? Maybe: `border: 0 none;` would be a better option?